fem/bilininteg.hpp

```diff
@@ -45,6 +45,15 @@ class BilinearFormIntegrator : public NonlinearFormIntegrator
        used later in the methods AddMultPA() and AddMultTransposePA(). */
    virtual void AssemblePA(const FiniteElementSpace &fes);
 
+   /// Method defining partial assembly.
+   /** The result of the partial assembly is stored internally so that it can be
+       used later in the methods AddMultPA() and AddMultTransposePA(). */
+   virtual void AssembleMixedPA(const FiniteElementSpace &trial_fes,
```

Comment:
Do we need "Mixed" in the name?  The arguments imply this.

Reply:
Just FYI: such method is also added in #1164 -- it'll be good to sync the two PRs.